src/components/sides/TaskInfo.vue

```diff
@@ -8,7 +8,7 @@
     ></div>
     <div class="side task-info">
       <action-panel
-        v-if="withActions && (!isConceptTask || selectedConcepts.size > 0)"
+        v-if="extendable && (!isConceptTask || selectedConcepts.size > 0)"
```

Comment:
Why this change?
The property `withActions` is designed to hide the action panel.
The property `extendable` is designed to allow to resize the parent panel.
Their behaviors are not linked.